### PR TITLE
CNV#50448: workaround KCS link for storage LM

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.17 only (RN)

Issue: [CNV-50448](https://issues.redhat.com//browse/CNV-50448)

Link to docs preview: https://84312--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-17-release-notes.html#virt-4-16-technology-preview_virt-4-17-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by djfjeff — APPROVED** [review]
LGTM

**Comment by peterclauterbach** [comment]
"follow the workaround documented"
technically not a workaround, it's a procedure to enable a feature gate.

**Comment by ousleyp** [comment]
> "follow the workaround documented" technically not a workaround, it's a procedure to enable a feature gate.

@peterclauterbach I hear you, but I was seeing it as a workaround because: 
- The procedure itself isn't supported by Red Hat, per the KCS.
- There would normally be a way to enable the feature gate in the HCO CR.

We tend to use "workaround" for things like this in the RNs. Thoughts? :)

**Comment by awels** [comment]
I agree with Pan, we are going to create a proper feature gate in HCO to enable this feature. So for now it is a workaround for the missing feature gate.
/lgtm

File: virt/release_notes/virt-4-17-release-notes.adoc [hunk 1]
```diff
@@ -163,6 +163,11 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 //CNV-40183 Release note: Tech preview for storage class migration
 * You can now migrate storage classes for running and stopped VMs.
++
+[NOTE]
+====
+Storage live migration is not enabled by default in the `HyperConverged` CR. To enable the required feature gates, follow the workaround documented in link:https://access.redhat.com/solutions/7089972[Enable storage live migration in {VirtProductName} 4.17] in the Red{nbsp}Hat knowledge base.
```
Comment (jeana-redhat) [inline virt/release_notes/virt-4-17-release-notes.adoc:169]: Not actually defined in this assembly
```suggestion
Storage live migration is not enabled by default in the `HyperConverged` custom resource. To enable the required feature gates, follow the workaround documented in link:https://access.redhat.com/solutions/7089972[Enable storage live migration in {VirtProductName} 4.17] in the Red{nbsp}Hat knowledge base.
```

**Review by jeana-redhat — COMMENTED** [review]
A lill baby nit :bow: 

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Review by kgoldbla — COMMENTED** [review]
looks good

**Pull request merged**